components/book/BookCard/BookCard.tsx

```diff
@@ -1,19 +1,37 @@
 import React from 'react'
 import Image from 'next/image'
-import { Book } from '../../../types'
+import Link from 'next/link'
+
+import { Book } from 'types'
+import { formatPrice } from 'utils/formatPrice'
+
+import styles from './BookCard.module.scss'
 
 const BookCard = ({ book }: { book: Book; eager: boolean }) => {
```

Comment:
what eager mean? it's not being used

Reply:
Removed [6f89538](https://github.com/thaoho21294/book-market-nextjs/pull/12/commits/6f895388b7869ec3c80395c5772533ec85b05ab2)
I forgot to remove it after copying it from the gatsby project.